### PR TITLE
add GPL2 license

### PR DESCRIPTION
**Pull request by ginakkiim**

### CODE REVIEW

**Comment by tyamaguchi-ucla** [comment]
I was looking at some examples but it looks like there's no need to modify the license at all? @ginakkiim @aholmes 

https://cran.r-project.org/web/packages/abctools/index.html

https://cran.r-project.org/web/licenses/GPL-2

**Comment by ginakkiim** [comment]
The repo/validation tool itself doesn't have any license at the moment, which is why I'm adding this license file for reference

**Comment by tyamaguchi-ucla** [comment]
Maybe `LICENSE.md`? I think it looks a bit better instead of `LICENSE`. Otherwise, looks good to me.

https://github.com/uclahs-cds/template-NextflowPipeline/blob/5deee773714a5024efba59bca0b23138d3901719/LICENSE.md

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,17 @@
+PipeVal is licensed under the GNU General Public License
```
Comment (aholmes) [inline LICENSE:1]: Swap the contents of LICENSE and COPYING. LICENSE is meant to contain the full license, while COPYING includes the license copyright clause and copying instructions.

Comment (ginakkiim) [reply LICENSE:1]: I think it's the other way around
https://www.gnu.org/licenses/gpl-howto.en.html
https://github.com/daviddao/CellProfiler-REST/tree/168992170aebe9925d70347d422163bd2ee5d4c0

Comment (aholmes) [reply LICENSE:1]: Good finds. Perhaps surprisingly, it seems [every variation](https://github.com/search?p=1&q=COPYING+LICENSE&type=Commits) under the sun is used. I generally reference [the Linux kernel](https://github.com/torvalds/linux), but even that is inconsistent with [Git](https://github.com/git/git) - Torvald's other major software project. I reference [dotnet](https://github.com/dotnet/runtime) and [Python](https://github.com/python/cpython) as well. So many differences!

Note that GitHub uses a Ruby Gem called [Licensee](https://github.blog/2016-09-21-license-now-displayed-on-repository-overview/), which looks specifically for `LICENSE` [files](https://github.com/licensee/licensee). Granted, the GitHub and GPL conventions are inconsistent. It's also worth noting that, when creating a new repository, if you select to add a license file, GitHub stores the contents in the file `LICENSE`.

So with all that information, it seems the decision is really ours. Personally I think we should follow the GitHub convention to stay consistent within our ecosystem.

Comment (ginakkiim) [reply LICENSE:1]: @aholmes Let me know if this is good to go!

**Review by aholmes — APPROVED** [review]

**Pull request merged**